guides/source/configuring.md

```diff
@@ -273,6 +274,10 @@ Forces all requests to be served over HTTPS, and sets "https://" as the default
 
 Sets the path where your app's JavaScript lives relative to the `app` directory. The default is `javascript`, used by [webpacker](https://github.com/rails/webpacker). An app's configured `javascript_path` will be excluded from `autoload_paths`.
 
+#### `config.log_file_size`
+
+Defines the maximum size of the Rails logger file for local and test environments.
```

Comment:
This doesn't quite match the implementation - if you configure a value, it will apply to all environments. Only the default value is environment specific.

```suggestion
Defines the maximum size of the Rails log file. Defaults to 100 MB in development and test, and unlimited in all other environments.
```

Reply:
this should look clearer